src/ai/backend/install/cli.py

```diff
@@ -226,9 +226,10 @@ def cancel_config(self) -> None:
 
 
 class InstallReport(Static):
-    def __init__(self, install_info: InstallInfo, **kwargs) -> None:
+    def __init__(self, install_info: InstallInfo, os_info, **kwargs) -> None:
```

Comment:
Let's add type hints to `os_info`

Reply:
Is it difficult to add type hints? @Yaminyam